ui/src/widgets/ui-slider/UISlider.vue

```diff
@@ -63,6 +86,27 @@ export default {
             msg.payload = this.value
             this.$store.commit('data/bind', msg)
             this.$socket.emit('widget-change', this.id, this.value)
+        },
+        onDynamicProperties (msg) {
+            const updates = msg.ui_update
+            if (!updates) {
+                return
+            }
+            if (updates.label) {
+                this.dynamic.label = updates.label
+            }
+            if (updates.thumbLabel) {
+                this.dynamic.thumbLabel = updates.thumbLabel
+            }
```

Comment:
checking `truthy` will prevent anyone clearing the dynamic value or setting an empty string.

I suggest a set of basic rules are adhered to across the board in all UI nodes so that there is one rule for users to get used to. 

Common tasks users do in dashboard:
1. Set an override - OK - possible
2. Set an empty string (e.g. no label) -  Not possible in this implementation (sending `""` would be ignored)
3. Clear an override - Not possible in this implementation

I would suggest, much like context, we differentiate missing property from `null`/`undefined` etc

The rule could be:
1. Exclude the property to skip updaing it (e.g. `ui_control: {}`)
2. Include the property to update it  (e.g. `ui_control: { label: xxx }`)
    1. `null` means remove any override
    2. anything else means use the override 

e.g. Take the case of setting a `label` 
* user sends `ui_control: { min: 7 }` - `label` is missing, dont update the dynamic label value!
* user sends `ui_control: { label: null }` - `label` is clearly present, use `null` to remove the dynamic property (and thus fall back to pre-set value)
* user sends `ui_control: { label: "" }` - `min` is clearly present, use its value (even if it is an empty string) to update the dynamic property

Reply:
in https://github.com/FlowFuse/node-red-dashboard/pull/864#discussion_r1618610128 you use !== undefined. Which means it would work as I described here:



> The rule could be:
> 
> 1. Exclude the property to skip updaing it (e.g. `ui_control: {}`)
> 2. Include the property to update it  (e.g. `ui_control: { label: xxx }`)
>    
>    1. `null` means remove any override
>    2. anything else means use the override

So my suggestion becomes:


```suggestion
            if (typeof updates.label !== 'undefined') {
                this.dynamic.label = updates.label
            }
            if (typeof updates.thumbLabel !== 'undefined') {
                this.dynamic.thumbLabel = updates.thumbLabel
            }
```

This way, since `null` already means "dont user the dynamic override", we can permit the user to send `""` for a deliberately empty label and `null` to undo the dynamic override.